src/screens/Test/TestPlace.tsx

```
@@ -21,43 +35,46 @@ const BodyByTab: Record<TabKey, React.ComponentType> = {
   stats: StatsScreen,
 };
 
-export default function TestPlace() {
+function MainTabContent({}: MainTabContentProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)